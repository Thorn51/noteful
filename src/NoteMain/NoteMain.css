.note_content {
  padding: 15px;
  line-height: 1.5rem;
}
