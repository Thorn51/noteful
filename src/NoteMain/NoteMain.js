import React from "react";
import moment from "moment";
import "./NoteMain.css";

export default function(props) {
  const note = props.notes.find(note => note.id === props.match.params.noteId);
  return (
    <section className="note_content">
      <div className="note">
        <h2 className="note_name">{note.name}</h2>
        <div className="note_modification">
          <p className="mod_date">
            Date modified on {""} {moment(note.modified).format("Do MMM YYY")}
          </p>
          <button className="delete_note">Delete Note</button>
        </div>
      </div>
      <div>
        <p className="note_content">{note.content}</p>
      </div>
    </section>
  );
}
