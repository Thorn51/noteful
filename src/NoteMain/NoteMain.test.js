import React from "react";
import ReactDOM from "react-dom";
import NoteMain from "./NoteMain";
import renderer from "react-test-renderer";

describe("NoteMain", () => {
  it("renders without throwing errors", () => {
    const div = document.createElement("div");
    ReactDOM.render(<NoteMain />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it("renders the component as expected", () => {
    const tree = renderer.create(<NoteMain />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
