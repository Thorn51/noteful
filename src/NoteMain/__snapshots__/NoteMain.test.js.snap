// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoteMain renders the component as expected 1`] = `
<p>
  Loading Notes...
</p>
`;
